src/lib.rs

```diff
@@ -92,6 +105,8 @@ pub enum Classification {
 #[serde(tag = "t", content = "c")]
 /// Location of an UAS, which may be relative to the CUAS.
 pub enum Location {
+    // All this variant descriptions are overwritten by the doc comments of their respective types
+
```

Comment:
What does this mean?

Reply:
This is more related to the schema visualizer and what description it shows when the mouse hovers over an element. See [this issue](https://github.com/COURAGEOUS-isf/format-website/issues/3).

Reply:
Remove it then. That's a visualizer issue. This should be completely independent of the visualizer.